cmd/ci-secret-generator/main_test.go

```diff
@@ -242,27 +290,33 @@ func TestExecuteCommand(t *testing.T) {
 			expected: []byte("basic case\n"),
 		},
 		{
-			name:          "error on no output",
-			cmd:           "true",
-			expectedError: fmt.Errorf("command \"true\" returned no output"),
+			name: "error on no output",
+			cmd:  "true",
+			expectedError: fmtExecCmdErr(execCmdValidateStdoutErrAction, "true",
```

Comment:
Could we display the error directly, instead of using `fmtExecCmdErr`?

Reply:
Sure